vendor/vendor.json

```diff
@@ -0,0 +1,15 @@
+{
+	"comment": "",
+	"ignore": "test",
+	"package": [
+		{
+			"checksumSHA1": "/kMYtkcxWV4dZNvJaEGkDUwt3TA=",
```

Comment:
Usually when I use vendoring, I try to vendor all the libraries...
I know it is an annoying job...